Heavy-Division/B78XH
# fix: Fix interactions.xml template

# Title
fix: removed default asobo template name from the interactions.xml





# Description 
This change alters the interactions.xml template to use a custom name rather than the default asobo template. This should fix issues that were previously causing the Gtn750 and other related products' screens to stop working. 
closes #176 



# Source material




# Testing Instructions 
navigate to this [url](https://github.com/nakajimayoshi/B78XH/tree/interactions-fix), click the code icon, download, and extract to your community folder (remove the current B78XH if it's there) 

If you have the C414 or any other GTN750 mod, ensure the screens and tablets are functioning by running the addon with a copy of the B78XH test build in your community folder.

Ensure the IRS align time is still normal, and no other screens or behaviors are broken


# Test Results




# Discord Username
HowNowBrownCrow#2591
